packages/shared/src/contexts/SettingsContext.tsx

```diff
@@ -180,17 +177,7 @@ export const SettingsContextProvider = ({
   };
 
   const syncSettings = async (bootUserId?: string) => {
-    const updatedSettings = settings;
-
-    if (settings.sidebarExpanded) {
-      const sidebarClosed = getFeatureValue(feature.sidebarClosed);
-
-      if (sidebarClosed) {
-        updatedSettings.sidebarExpanded = false;
-      }
-    }
-
-    await updateRemoteSettingsFn(updatedSettings, bootUserId);
+    await updateRemoteSettingsFn(settings, bootUserId);
```

Comment:
With `false` being new default we should probably also look at the layout shift on post pages and decide which state we want supported there. Since now the padding during SSR expects menu to be there. cc: @rebelchris you will remember this as well when you optimized SSR there

Reply:
Yeah true, I remember that discussion. Let me take a look

Reply:
Previous thread where this was discussed https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1714294470698459

Reply:
As long as we make sure everyone knows what will happen now for new users I am fine with it 👍